src/UI/Settings.js

```diff
@@ -918,7 +919,11 @@ function Settings(props) {
         </ReactTooltip>
         <Form onSubmit={handleSubmit}>
           <H3>Change primary color</H3>
-          <Input placeholder="Hex or string" ref={primaryColorInput} />
+          <Input
+            defaultValue={theme.primary_color}
```

Comment:
You should not rely on the theme, instead rely on the useTheme() object that has access to all the theme attributes like this:
useTheme().primary_color
You have multiple examples in this file to see how it's used.